### PR TITLE
CAFF-55  - tuning navigation parameters (minheights, local costmap size)

### PR DESCRIPTION
**Pull request by MarcM0**
Changing values, looking at obstacles in rviz, until numbers were fine tuned

### CODE REVIEW

File: nav/nav_stack/config/costmap_common_params.yaml [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
+
   marking: true, 
-  clearing: true,
-  obstacle_range: 15.0,  # 15.0,
-  raytrace_range: 16.0   # 16.0
-}
+  clearing: true, #doesn't point cloud doesnt see most of ramp, so it clears it
+  obstacle_range: 15.0, # 15.0,
+  raytrace_range: 16.0 # 16.0
+}
```
Comment (1999michael) [inline nav/nav_stack/config/costmap_common_params.yaml:33]: Add newline to end of file

File: nav/nav_stack/config/local_costmap_params.yaml [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
-  resolution: 0.05
+global_frame: caffeine/odom
+robot_base_frame: caffeine/base_link
+update_frequency: 5.0
+publish_frequency: 2.0
+static_map: false
+rolling_window: true
+width: 8.0
+height: 8.0
+resolution: 0.05
```
Comment (1999michael) [inline nav/nav_stack/config/local_costmap_params.yaml:10]: Add newline to end of file

**Review by 1999michael — CHANGES_REQUESTED** [review]
End of file needs empty lines, otherwise it looks good to me

File: nav/nav_stack/config/costmap_common_params.yaml (hunk 1 above)
Comment (MarcM0) [reply nav/nav_stack/config/costmap_common_params.yaml]: added

File: nav/nav_stack/config/local_costmap_params.yaml (hunk 2 above)
Comment (MarcM0) [reply nav/nav_stack/config/local_costmap_params.yaml]: added

**Review by 1999michael — APPROVED** [review]
Looks good to me

**Review by erickmu1 — APPROVED** [review]

**Pull request merged**